notebooks/cytomining-ecosystem/visualize-landscape-fall-2023.py

```diff
@@ -0,0 +1,688 @@
+    ),
+)
+
+fig_collection.append(
+    {
+        "plot": fig_languages,
+        "description": """
+        This plot shows the primary programming language project count colored by category.
+        A primary programming language is derived from the maximum lines of code from GitHub-detected programming languages.
+        """,
+        "findings": """
+        We observe that R, Python, and Jupyter Notebooks are among the most popular primary programming languages found
+        within the Cytomining Ecosystem software landscape.
+        """,
+        "section": "General landscape",
+    }
+)
+
+# Show the plot
+fig_languages.show()
+
+# +
+# gather project org data
+df_project_orgs = df_projects["GitHub Organization"].value_counts()
+
+# Create a horizontal bar chart using Plotly Express
+fig_orgs = px.bar(
+    df_project_orgs[df_project_orgs > 1].sort_values(ascending=True),
+    orientation="h",
+    width=1200,
+    height=1000,
+)
+
+fig_orgs.update_layout(
+    title="GitHub Organization Project Count (filtered to project count >= 2)",
+    xaxis_title="Projects Count",
+    showlegend=False,
+    yaxis=dict(
+        tickmode="array",
+        tickvals=df_project_orgs.index.tolist(),
+        ticktext=df_project_orgs.index.tolist(),
+    ),
+)
+
+fig_collection.append(
+    {
+        "plot": fig_orgs,
+        "description": """
+        This plot shows the count of projects per GitHub organization (a collection of repositories)
+        for those organizations with more than 2 projects.
+        """,
+        "findings": """
+        Here we find that the Theis Lab, UC Davis Bioinformatics Training, and Teich Lab are at the top of this listing.
+        """,
+        "section": "General landscape",
+    }
+)
+
+# Show the plot
+fig_orgs.show()
+
+# +
+# gather project org data
+df_project_orgs_stars = (
+    df_projects[df_projects["category"] != "relevant-open-source"]
+    .groupby(by=["GitHub Organization"])["GitHub Stars"]
+    .sum()
+)
+
+# Create a horizontal bar chart using Plotly Express
+fig_orgs_stars = px.bar(
+    df_project_orgs_stars[df_project_orgs_stars > 100].sort_values(ascending=True),
+    orientation="h",
+    width=1200,
+    height=1000,
+)
+
+fig_orgs_stars.update_layout(
+    title="GitHub Organization Star Count (filtered to project count > 100)",
+    xaxis_title="GitHub Stars",
+    showlegend=False,
+    yaxis=dict(
+        tickmode="array",
+        tickvals=df_project_orgs_stars.index.tolist(),
+        ticktext=df_project_orgs_stars.index.tolist(),
+    ),
+)
+
+fig_collection.append(
+    {
+        "plot": fig_orgs_stars,
+        "description": """
+        This plot shows the sum of GitHub stars per GitHub organization for those with a sum total of stars greater than 100.
+        """,
+        "findings": """
+        Here we can observe that scverse, Theis Lab, and Satija Lab organizations hold the top count of GitHub stars.
```

Comment:
Is there one sentence to add about Cytomining? It is nice to keep this focus in the findings section

Reply:
Thanks, added a statement: `The Cytomining organization is found near the middle of this visualization, indicating similar popularity and early success in audience outreach amidst those within the related landscape.`